Source/Details/_ASDisplayLayer.mm

```diff
@@ -179,13 +138,14 @@ + (id)defaultValueForKey:(NSString *)key
 {
   if ([key isEqualToString:@"displaysAsynchronously"]) {
     return @YES;
+  } else if ([key isEqualToString:@"opaque"]) {
```

Comment:
I'm not very familiar with the semantics of defaultValueForKey when used with standard layer properties like this. That said:

- I think having the code in init is more visible for a user exploring where a potentially-unexpected default opacity is coming from.
- This implementation will perform isEqualToString: on potentially thousands of accesses to unrelated properties. Although unlikely to be a focused performance hotspot, it probably would create distributed cost.
- What do you think about removing this method entirely and setting both self.opaque = YES and self.displaysAsynchronously = YES in the init method?